packages/pwa-kit-runtime/src/ssr/server/build-remote-server.js

```diff
@@ -595,6 +598,48 @@ export const RemoteServerFactory = {
         })
     },
 
+    /**
+     * @private
+     */
+    _setupSlasPrivateClientProxy(app, options) {
+        if (options.useSlasPrivateClient) {
+            const clientSecret = process?.env?.SLAS_PRIVATE_CLIENT_SECRET
+            if (!clientSecret) {
+                app.use('/ssr/auth', (_, res) => {
+                    return res.status(501).json({
+                        message:
+                            'Environment variable SLAS_PRIVATE_CLIENT_ID not set: LINK_TO_DOC HERE'
+                    })
+                })
+            } else {
+                const clientId = options.mobify.app.commerceAPI.parameters.clientId
+                const shortCode = options.mobify.app.commerceAPI.parameters.shortCode
+
+                const slasTarget = `https://${shortCode}.api.commercecloud.salesforce.com`
+
+                app.use(
+                    '/ssr/auth',
+                    createProxyMiddleware({
+                        target: slasTarget,
+                        changeOrigin: true,
+                        pathRewrite: {'/ssr/auth': ''},
+                        onProxyReq: (outGoingReq, incomingReq) => {
+                            if (incomingReq.path.match(/\/oauth2\/token/)) {
+                                const encodedSlasCredentials = Buffer.from(
+                                    `${clientId}:${clientSecret}`
+                                ).toString('base64')
```

Comment:
Not a big deal, but I'd move this outside of the function so it is calculated once.